web/ui/react-app/src/contexts/PathContexts.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Now that there is only one path context in here, how about renaming `PathContexts.tsx` to `PathPrefixContext.tsx`?